source/fuzz/fuzzer_util.h

```diff
@@ -23,13 +24,45 @@
 #include "source/opt/basic_block.h"
 #include "source/opt/instruction.h"
 #include "source/opt/ir_context.h"
+#include "spirv-tools/libspirv.hpp"
 
 namespace spvtools {
 namespace fuzz {
 
 // Provides types and global utility methods for use by the fuzzer
 namespace fuzzerutil {
 
+namespace {
```

Comment:
I'm not sure if we actually need an anonymous namespace here. I'm not against it, though.

Reply:
Yeah we should probably define these constants once in the cpp file and then declare them extern here without an anonymous namespace. But I see that kConsoleMessageConsumer is used in tests...maybe only in tests? So that could be moved to fuzzer_test_util.

Reply:
Thanks - I have addressed this in my latest push.